src/components/App/App.tsx

```diff
@@ -0,0 +1,10 @@
+import SearchPage from "../SearchPage";
```

Comment:
from my perspective your index.tsx is already the app file, I don't see any benefits in having separate app.tsx file with header and searchpage components